src/store.rs

```diff
@@ -12,46 +12,46 @@ pub struct Store {
 impl Store {
     pub fn new(url: &str, expire_keys: u64)
                -> Result<Store, String> {
-        let client = try!(redis::Client::open(url).map_err(stringify_err));
+        let client = try!(redis::Client::open(url).map_err(Self::stringify_err));
         Ok(Store { client: client, expire_keys: expire_keys })
     }
 
     pub fn store_session(&self, session_id: &str, data: &[(&str, &str)])
                          -> Result<(), String> {
-        let key = format_session_key(session_id);
-        try!(self.client.hset_multiple(key.as_str(), data).map_err(stringify_err));
-        try!(self.client.expire(key.as_str(), self.expire_keys as usize).map_err(stringify_err));
+        let key = Self::format_session_key(session_id);
+        try!(self.client.hset_multiple(key.as_str(), data).map_err(Self::stringify_err));
+        try!(self.client.expire(key.as_str(), self.expire_keys as usize).map_err(Self::stringify_err));
         Ok(())
     }
 
     pub fn get_session(&self, session_id: &str)
                        -> Result<HashMap<String, String>, String> {
-        let key = format_session_key(session_id);
-        self.client.hgetall(key.as_str()).map_err(stringify_err)
+        let key = Self::format_session_key(session_id);
+        self.client.hgetall(key.as_str()).map_err(Self::stringify_err)
     }
 
     pub fn store_cache(&self, url: &str, data: &str, seconds: usize)
                        -> Result<(), String> {
-        let key = format_cache_key(url);
-        try!(self.client.set_ex(key.as_str(), data, seconds).map_err(stringify_err));
+        let key = Self::format_cache_key(url);
+        try!(self.client.set_ex(key.as_str(), data, seconds).map_err(Self::stringify_err));
         Ok(())
     }
 
     pub fn get_cache(&self, url: &str)
                      -> Result<Option<String>, String> {
-        let key = format_cache_key(url);
-        self.client.get(key.as_str()).map_err(stringify_err)
+        let key = Self::format_cache_key(url);
+        self.client.get(key.as_str()).map_err(Self::stringify_err)
     }
-}
 
-fn stringify_err(err: RedisError) -> String {
-    err.description().to_string()
-}
+    fn stringify_err(err: RedisError) -> String {
```

Comment:
Squash this into the commit that introduces the function, please (only if the `From`/`Into` implementation doesn't work out; that should also be part of the same commit).